source/grid/grid_generator.cc

```diff
@@ -3111,7 +3112,7 @@ namespace GridGenerator
                   = cell->face(i);
 
                 const Point<3> face_center (face->center());
-                if (std::abs((face_center(0)-center(0)) > 1.e-3))
+                if (std::abs(face_center(0)-center(0)) > 1.e-6)
                   {
```

Comment:
Am 07. Mai 2015, 10:12 schrieb Martin Kronbichler <notifications@github.com>:

> In source/grid/grid_generator.cc:
>
>> @@ -3111,7 +3112,7 @@ namespace GridGenerator
>>                    = cell->face(i);
>>  
>>                  const Point<3> face_center (face->center());
>> -                if (std::abs((face_center(0)-center(0)) > 1.e-3))
>> +                if (std::abs(face_center(0)-center(0)) > 1.e-6)
>>                    {
>
> Any reason to also change the tolerance? Approved for merge
> otherwise.

We usually have "1.e-6" as magic number and not 1.e-3 which is a bit
large, i.e., a 2d grid with 1 million cells of the unit square already
reaches a face size of 1.e-3.

Which reminds me of the fact that we have to come up with a sane
solution for such magic numbers eventually...